modules/apps/frontend-js/frontend-js-walkthrough-web/src/main/resources/META-INF/resources/Walkthrough.js

```diff
@@ -310,7 +318,9 @@ const Step = ({
 
 	useObserveRect(align, popoverRef?.current);
 
-	const currentLayoutRelativeURL = themeDisplay.getLayoutRelativeURL();
+	const currentLayoutRelativeURL = themeDisplay
+		.getLayoutRelativeURL()
+		.replace(SITE_PREFIX_PATH, '');
```

Comment:
This `.replace` was necessary because we found cases where `/web/guest` on admin are being added on layoutRelativeURL